src/main/java/seedu/address/model/Model.java

```diff
@@ -91,7 +92,7 @@ public interface Model {
 
     //=========== Student ================================================================================
 
-
+    Student getStudentByNumber(StudentNumber studentNumber);
```

Comment:
This will cause some compilation within tests made prior to this. So do help ensure some of the tests now implement this method

Reply:
I've removed this at the moment because we don't need it for our product MVP